src/loaders/ObjectLoader.js

```diff
@@ -550,13 +583,29 @@ Object.assign( ObjectLoader.prototype, {
 					break;
 
 				case 'Mesh':
+				case 'SkinnedMesh':
 
 					var geometry = getGeometry( data.geometry );
 					var material = getMaterial( data.material );
+					var skeleton = getSkeleton( data.skeleton );
+
+					// override bones if skeleton exists
+					if ( skeleton !== undefined ) {
+
+						geometry.bones = skeleton.bones;
+						geometry.boneInverses = skeleton.boneInverses;
+
+					}
 
 					if ( geometry.bones && geometry.bones.length > 0 ) {
 
-						object = new SkinnedMesh( geometry, material );
+						var useVertexTexture = ( skeleton !== undefined ) ? skeleton.useVertexTexture : undefined;
+
+						object = new SkinnedMesh( geometry, material, useVertexTexture );
+
+						if ( data.bindMode !== undefined ) object.bindMode = data.bindMode;
+						if ( data.bindMatrix !== undefined ) object.bindMatrix.fromArray( data.bindMatrix );
+						object.updateMatrixWorld( true );
```

Comment:
It would suit a use case of mine to be able to load SkinnedMesh types directly;

I have a character model made up of multiple SkinnedMesh types representing various body parts and clothing items.
If I have to specify bones for each SkinnedMesh part, they'd all be duplicate bones for the whole skeleton.

I would still like a way to have multiple skinned meshes share a skeleton and be loadable via the ObjectLoader, and I think having the SkinnedMesh type be loaded directly for objects without a .bones property would help.